docs/wip/what-is-gatsby.md

```diff
@@ -1,13 +1,9 @@
-We expect that this user already knows what is Jamstack. If not, he must be redirected to our "What is Jamstack" doc. At the end of this doc, the reader must understand what Gatsby is and its main concepts. He must have enough understanding (or know where he can learn more) to feel comfortable to advance to our docs on how to develop a storefront using VTEX and Gatsby.
+# What is Gatsby?
 
-Objective:
+GatsbyJS é um gerador de sites estático moderno baseado em React e GraphQL. Que torna a experiencia do desenvolvedor muito agradável por usar as melhores partes do React, Webpack, GraphQL e outras ferramentas.
 
-Briefly explain what is Gatsby and its main concepts
+## Por que o escolhemos?
 
-This explanation doesn't need to be a long one. Instead, we must redirect the user to official resources.
+A maneira como Gatsby usa GraphQL é feito para coletar seus dados de qualquer lugar: Markdown, JSON, seu CMS favorito, APIs de terceiros, em qualquer lugar! E no momento da construção, ele cria um servidor GraphQL interno de todos esses dados. Portanto, em seus componentes React seus dados são consultados em tempo de construção daquele mesmo lugar, da mesma forma por meio do GraphQL. Além da forma em como o GraphQL é utilizado, o gatsby é poderoso pelo seu ecossistema baseado em plugins, onde pode-se encontrar milhares deles em sua [página](https://www.gatsbyjs.com/plugins).
 
-When talking about Gatsby concepts, we can briefly explain how we use them in the Faststore specific context.
-
-Explain why we chose Gatsby and its main benefits.
-
-Organize it into topics followed by a brief explanation.Example: Flexibility to.. Blalblalbla
+Além de podermos nos alavancar da arquitetura de plugins, também podemos aumentar nosso nível de personalização a partir de shadowing. Este recurso nos permite substituir um arquivo do `src` incluído no pacote webpack com seu próprio arquivo.
```

Comment:
## Why we chose Gatsby?

### No breaking changes

Most technologies used by Gatsby are already familiar to the developers working with VTEX IO. The same way as with Gatsby, our [Store Framework components](https://developers.vtex.com/vtex-developer-docs/docs/what-is-store-framework) are built with React. Bringing Gatsby and VTEX together means a fast learning curve to a universe that will allow you to build the fastest stores in the market.

### Easy data integration

Leveraging from the GraphQL power, Gatsby allows you to collect data from any source of your choosing: Markdown, JSON, third-party APIs, and even from the VTEX CMS.  
By creating an internal GraphQL server, Gatsby makes all that fetched data accessible via React components during build time. 

### Extensible architecture

Gatsby is made more flexible and extensible thanks to its plugin-based ecosystem. Thousands of plugins can be found on the Gatsby [website] (https://www.gatsbyjs.com/plugins). When diving deeper into our new way of building stores, you'll find out more about VTEX plugins. 

### High customization level

With [Shadowing](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/), Gatsby increases customization levels, allowing the user to replace [webpack](https://webpack.js.org/) files with customized ones.